doc/classes/Basis.xml

```diff
@@ -190,6 +243,18 @@
 			<return type="Basis" />
 			<description>
 				Returns the transposed version of the matrix.
```

Comment:
```suggestion
				Returns the transposed version of the matrix, this turns rows into columns and vice versa.
```
I think having a verbal description of what this means along with the example would be good, wording is just a suggestion

Reply:
I was actively avoiding to touch the descriptions in this PR. I have always gone too far and it's part of the reason as too why all of the past "Overhaul Documentation" PRs have been taking so long to be merged.

Reply:
Then let's do that in separately 🙂